Patches/[O21] Outer Rim/Patch_OuterRim_Apparel.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>Outer Rim - Tatooine (Continued)</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			
+		<li Class="PatchOperationAdd">
+			<xpath>Defs/ThingDef[defName="O21_OuterRim_JawaRobe" or defName="O21_OuterRim_TuskenRaiderRobeA" or defName="O21_OuterRim_TuskenRaiderRobeB"]</xpath>
+			<value>
+				<equippedStatOffsets>
+				  <CarryBulk>60</CarryBulk>
+				</equippedStatOffsets>
+			</value>
+		</li>
+		
+		<li Class="PatchOperationReplace">
+			<xpath>Defs/ThingDef[defName="O21_OuterRim_JawaRobe" or defName="O21_OuterRim_TuskenRaiderRobeA" or defName="O21_OuterRim_TuskenRaiderRobeB"]/statBases/StuffEffectMultiplierArmor</xpath>
+			<value>
+				<StuffEffectMultiplierArmor>10</StuffEffectMultiplierArmor>
+			</value>
+		</li>
+		
+		<li Class="PatchOperationAdd">
+			<xpath>Defs/ThingDef[defName="O21_OuterRim_JawaRobe" or defName="O21_OuterRim_TuskenRaiderRobeA" or defName="O21_OuterRim_TuskenRaiderRobeB"]/statBases</xpath>
+			<value>
+				<Bulk>7.5</Bulk>
+				<WornBulk>1</WornBulk>
```

Comment:
These are full-body robes, so the wornBulk can be set to 1.5 - 2, since it's a fairly large garment.

Reply:
Done